mypy.ini

```diff
@@ -1,10 +1,11 @@
 [mypy]
 python_version = 3.9
 pretty = true
+mypy_path = ./type_stubs
 exclude = (?x)(
-          docs|
-          \.run|
-          \.tox
+          /docs/
+          | /\.run/
+          | /\.tox/
           )
```

Comment:
This corrects the paths to exclude when doing static type checking with mypy.

Reply:
Independent of the new stub file, correct?

Reply:
Correct, this is an independent change.